openshift/s2i/bin/run

```diff
@@ -97,11 +97,7 @@ if is_gunicorn_installed; then
     export WEB_CONCURRENCY=${WEB_CONCURRENCY:-$(get_default_web_concurrency)}
 
     echo "---> Serving application with gunicorn ($APP_MODULE) ..."
-    if [ $CLOWDER_ENABLED == "true" ]; then
-      exec gunicorn "$APP_MODULE" --chdir=$APP_HOME --bind=0.0.0.0:8000 --config "$APP_CONFIG" --preload
-    else
-      exec gunicorn "$APP_MODULE" --chdir=$APP_HOME --bind=0.0.0.0:8080 --config "$APP_CONFIG" --preload
-    fi
+    exec gunicorn "$APP_MODULE" --chdir=$APP_HOME --bind=0.0.0.0:8080 --config "$APP_CONFIG" --preload
```

Comment:
@redallen - Curious why these conditionals were removed? We'll still need different behavior for Clowder for port `8080` vs `8000`.

Reply:
Ah sorry I didn't notice the port difference.

I removed them because the bash syntax is incorrect if `CLOWDER_ENABLED` is undefined and the script was always throwing an error. I'll add it back and quote it:)

Reply:
Should be fixed, please rereview @coderbydesign !